src/NsisoLauncherCore/Util/AssetsReader.cs

```diff
@@ -44,16 +44,18 @@ public JAssets GetAssets(Modules.Version version)
 
     public class JAssets
     {
-        [JsonProperty("objects")]
-        public Dictionary<string, JAssetsInfo> Objects { get; set; }
+        [JsonProperty("objects")] public Dictionary<string, JAssetsInfo> Objects { get; set; }
```

Comment:
Github action's warning:
Change 'Objects' to be read-only by removing the property setter.